drs-auth_client/lib/drs/auth_client/client.rb

```diff
@@ -13,12 +13,13 @@ def initialize(auth_token)
         @auth_token = auth_token
       end
 
-      def get(path)
+      def get(path, options = {})
```

Comment:
This may be a bit pedantic, but I'd call the array `params`, because it's passed straight to the request as params. `options` for me indicates that I can configure the client or request further. 

But I'm not insisting on this, if you think `options` is more appropriate.